mrp_bom_structure_report_level_1/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+================================
+MRP BOM Structure Report Level 1
+================================
+
+This module adds a PDF report in BOM's to print just the first level of
+components. So, will not explode into the subcomponents.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to 'Manufacturing / Products / Bill of Materials'
```

Comment:
Instead of 'Manufacturing / Products' I see 'Manufacturing / Master Data':

![image](https://user-images.githubusercontent.com/28435599/31073290-91d27e30-a76b-11e7-87e5-dda8277e5f9f.png)